content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -142,6 +142,22 @@ For information about enabling IPVS mode with kubeadm see:
 For information about passing flags to control plane components see:
 - [control-plane-flags](/docs/setup/production-environment/tools/kubeadm/control-plane-flags/)
 
+### Running kubeadm without an internet connection {#without-internet-connection}
+
+For running kubeadm without an internet connection you have to pre-pull the required control-plane images.
+
+You can list and pull the images using the `kubeadm config images` sub-command:
+
+```shell
+kubeadm config images list
+kubeadm config images pull
+```
+
+You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file).
```

Comment:
```suggestion
You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file)
```

Reply:
Hm, why remove the dot at the end?

Reply:
Emm...  it is not a complete sentence, see the next line.
```
You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file).
to control the `kubernetesVersion` and `imageRepository` fields.
```

Reply:
Oops i was looking at these pr comments on mobile.